src/main.js

```diff
@@ -432,6 +432,10 @@ function handleAppWebContentsCreated(dc, contents) {
       log.info(`Untrusted popup window blocked: ${url}`);
       return;
     }
+    if (isKnownTeam(url)) {
+      log.info('this is a known team, preventing to open a new window');
+      return;
```

Comment:
Just curious, does this stop the link from being opened altogether?

Reply:
no, it still opens in the browser